packages/manager/src/features/LoadBalancers/LoadBalancerCreate/LoadBalancerConfiguration.tsx

```diff
@@ -0,0 +1,55 @@
+import Stack from '@mui/material/Stack';
+import { useTheme } from '@mui/material/styles';
+import * as React from 'react';
+
+import { Paper } from 'src/components/Paper';
+import { Typography } from 'src/components/Typography';
+import { VerticalLinearStepper } from 'src/components/VerticalLinearStepper/VerticalLinearStepper';
+
+export const configurationSteps = [
+  {
+    content: <div>TODO: Implement Details step content.</div>,
+    handler: () => null,
+    label: 'Details',
+  },
+  {
+    content: <div>TODO: Implement Service Targets Configuration.</div>,
+    handler: () => null,
+    label: 'Service Targets',
+  },
+  {
+    content: <div>TODO: Implement Routes Confiugataion.</div>,
+    handler: () => null,
+    label: 'Routes',
+  },
+];
+
+export const LoadBalancerConfiguration = () => {
+  const theme = useTheme();
+
+  return (
+    <Paper
+      sx={{
+        flexGrow: 1,
+        marginTop: theme.spacing(3),
+        width: '100%',
+      }}
```

Comment:
In this way we can get rid of the useTheme hook (+ the sx below already does it this way)
```suggestion
      sx={(theme) => ({
        flexGrow: 1,
        marginTop: theme.spacing(3),
        width: '100%',
      })}
```